### PR TITLE
Removing privacy day content.

### PR DESCRIPTION
**Pull request by simonwex**
Fixes https://github.com/MozillaFoundation/plan/issues/352

### CODE REVIEW

**Comment by simonwex** [comment]
Note: I didn't remove the private-eye route, since I'm not sure if any external parties are linking to it.

**Comment by HPaulJohnson** [comment]
@simonwex just to clarify, all external sources are linking directly to /private-eye URL so we should also remove the "Get Smart on Privacy" / learn more promo in the middle of the page as well. Not sure if that's what you meant but as long as the /private-eye page exists we are OK to remove all other privacy content from home page.

File: less/angular/pages/home.less [hunk 1]
```diff
@@ -528,502 +528,3 @@
...
-// Privacy day!
-
-@privacyred: #dd4d58;
-@privacygrey: #efefef;
-
-.home-privacy-day {
-  background-color: darken(@white, 6.25%);
-  background-size: 30rem;
-  @media @standard-def {
-    background-image: url('/img/home/background-privacy.png');
```
Comment (jbuck) [inline less/angular/pages/home.less]: This file can be deleted

File: less/angular/pages/home.less [hunk 2]
```diff
@@ -528,502 +528,3 @@
...
-  }
-  @media @retina {
-    background-image: url('/img/home/background-privacy@2x.png');
```
Comment (jbuck) [inline less/angular/pages/home.less]: This file can be deleted

File: public/views/home-3.html [hunk 3]
```diff
@@ -28,71 +28,6 @@
...
-                    &nbsp;
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </figure>
-      </a>
-
-      <h2>{{ 'Get smart on privacy' | i18n }}</h2>
```
Comment (jbuck) [inline public/views/home-3.html]: This string can be deleted from the locale file

File: public/views/home-3.html [hunk 4]
```diff
@@ -28,71 +28,6 @@
...
-
-      <h4>
-        {{ 'Understanding your privacy on the web' | i18n }}
```
Comment (jbuck) [inline public/views/home-3.html]: This string can be deleted from the locale file

**Comment by jbuck** [comment]
@simonwex a small amount of left-over files and strings to clean up, r+ when that's done

**Comment by simonwex** [comment]
@jbuck could you get this pushed out when you have a chance?

**Comment by jbuck** [comment]
@simonwex done

**Comment by simonwex** [comment]
Thanks, @jbuck!

**Pull request merged**